src/dbus/org.levak.Warfacebot.xml

```diff
@@ -56,6 +56,7 @@
     </method>
     <method name="BuddyAdd">
       <arg name="Nickname" type="s" direction="in" />
+      <arg name="ErrorCode" type="a{sv}" direction="out" />
```

Comment:
Do you really need a dictionary being returned?
Suggesting to just return an integer.